src/vs/base/test/browser/ui/tree/asyncDataTree.test.ts

```diff
@@ -318,8 +318,8 @@ suite('AsyncDataTree', function () {
 		assert.equal(aNodeUpdated1.children.length, 0);
 		let didCheckNoChildren = false;
 		const event = tree.onDidChangeCollapseState(e => {
-			const child = container.querySelector('.monaco-list-row:nth-child(2)') as HTMLElement | undefined;
-			assert.equal(child, undefined);
+			const child = container.querySelector('.monaco-list-row:nth-child(2)') as HTMLElement | null;
+			assert.equal(child, null);
```

Comment:
This actually changes runtime behavior of tests, I'm surprised they still pass. It tells me this line of code isn't running at all.